preferences/src/main/java/dev/jdtech/jellyfin/AppPreferences.kt

```diff
@@ -34,6 +34,7 @@ constructor(
     val theme get() = sharedPreferences.getString(Constants.PREF_THEME, null)
     val dynamicColors get() = sharedPreferences.getBoolean(Constants.PREF_DYNAMIC_COLORS, true)
     val amoledTheme get() = sharedPreferences.getBoolean(Constants.PREF_AMOLED_THEME, false)
+    val spanCount get() = sharedPreferences.getInt(Constants.PREF_SPAN_NO, 2)
```

Comment:
Don't hardcode the value here, the `dimens.xml` file contains a `library_columns` integer value which is used for different screen sizes as default.

https://github.com/jarnedemeulemeester/findroid/blob/5aad3781320ab7f989c54cd48a8638b3b88aa72c/core/src/main/res/values/dimens.xml#L9

Also, how about adding this number preference to the kebab menu? I think it will be better there!

Reply:
Thank you! 
I didn't know of the integer value in `dimens.xml`, my bad!
Also, by kebab menu, you mean the 3 dots in the upper right corner, in each library? 
If that's what you mean, I didn't add it there because, since there are multiple libraries, I think it makes more sense for it to be a global settings (Although I don't think it would necessarily be a bad idea to make it a per-library setting).

Reply:
Yes, I meant the three dot menu. 

Also, we can keep only one single preference instead of per library since there may be too many of them, it'll just be more convenient to switch it without leaving the library in my opinion.

Reply:
I agree with you, but I'm assuming that app only has one library, so changing it there makes more sense.

Since Findroid can have multiple libraries, I don't think having a per-library setting to do this makes much sense, since it would only complicate things for no real reason. 

Also, especially on devices with larger screens (Tablets and especially TVs), having only 2 covers shown, regardless of the number of items in a given library, isn't really ideal, since you'd end up with massive cover images that take up the whole screen.

Thanks for the input, though!

Reply:
Yup, maybe we can add landscape and portrait mode seekbar like in the screenshot above, in settings only. It will be better I think for both orientations.

Reply:
> Yup, maybe we can add landscape and portrait mode seekbar like in the screenshot above, in settings and use it accordingly with respect to the orientation. It will be better I think.

I'm sorry but I'm not sure I understand what you mean.
We should add a seekbar that shows up in the kebab menu when the app is in landscape mode?

Reply:
> > Yup, maybe we can add landscape and portrait mode seekbar like in the screenshot above, in settings and use it accordingly with respect to the orientation. It will be better I think.
> 
> I'm sorry but I'm not sure I understand what you mean. We should add a seekbar that shows up in the kebab menu when the app is in landscape mode?

No, I meant a seekbar which changes both portrait and landscape values at once like in the screenshot `Portrait: 4 - Landscape: 6`

Reply:
> > > Yup, maybe we can add landscape and portrait mode seekbar like in the screenshot above, in settings and use it accordingly with respect to the orientation. It will be better I think.
> > 
> > I'm sorry but I'm not sure I understand what you mean. We should add a seekbar that shows up in the kebab menu when the app is in landscape mode?
> 
> No, I meant a seekbar which changes both portrait and landscape values at once like in the screenshot `Portrait: 4 - Landscape: 6`, since we would want more rows while in landscape mode.

Oh, so individual preferences for each screen orientation.

I like the idea, but I don't think seekbars would be appropriate for this, as it would look quite odd.

Also, another concern I have is that it would complicate things for the end user quite a lot more.